src/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -0,0 +1,143 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+import type { ReactNode } from "react";
+
+export type DropdownOptionPros = {
+  label: string;
+  value: string | number;
+  selected?: boolean;
+  disabled?: boolean;
+};
+
+export type CodeSnippetDropdownProps = {
```

Comment:
Fair point, I guess it was just copy/paste - I was basing it on implementation in some other component.

Reply:
We've had quite a few cases where we end up wrapping a react-components component in maas-ui and want to pass the props to the inner component. We've had to go back and export the props in react-components so that we don't have to redefine them in maas-ui, so I think the pattern should be to always export the type for the props on react-components.